benchmarks/templates/benchmarks/components/left-sidebar.html

```diff
@@ -1,4 +1,18 @@
-<a class="selected" href="#">Leaderboard</a>
-<a href="#">Compare</a>
-<a href="#">Tutorials</a>
-<a href="#">Profile</a>
+{% load static %}
+<a href="http://{{ request.get_host }}/2023"><img class="brainscore-logo" src="{% static "/benchmarks/img/b-w_logo.png" %}" alt="Brain-Score Logo"></a>
+<a class="selected" href="http://{{ request.get_host }}/{{ domain }}/">
+	<img src="{% static "/benchmarks/img/leaderboard_icon.png" %}" alt="Brain-Score Logo">
+	<span>Leaderboard</span>
+</a>
+<a href="http://{{ request.get_host }}/{{ domain }}/#compare">
+	<img src="{% static "/benchmarks/img/compare_icon.png" %}" alt="Brain-Score Logo">
+	<span>Compare</span>
+</a>
+<a href="http://{{ request.get_host }}/{{ domain }}/#tutorial">
+	<img src="{% static "/benchmarks/img/tutorial_icon.png" %}" alt="Brain-Score Logo">
+	<span>Tutorials</span>
```

Comment:
This can point to www.brain-score.org/tutorial (tutorials are on a separate PR but will be merged soon, so this will link them up)